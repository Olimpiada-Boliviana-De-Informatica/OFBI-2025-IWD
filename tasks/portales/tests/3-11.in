22
151561800 10650478 370965174 116244626 759232427 120850316 687686381 45890490 87604552 850020697 846333655 988975984 584950657 722277356 707539910 250775052 543942075 990924031 664671826 328900298 341177567 365896559