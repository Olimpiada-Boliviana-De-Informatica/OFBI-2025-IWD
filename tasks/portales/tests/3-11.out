-1
-1
151561800
10650478
370965174
116244626
370965174
10650478
45890490
759232427
759232427
850020697
370965174
687686381
687686381
151561800
370965174
988975984
584950657
250775052
328900298
341177567
